modules/dataverse-parent/pom.xml

```diff
@@ -147,10 +147,10 @@
     
         <!-- Major system components and dependencies -->
         <payara.version>5.2021.6</payara.version>
-        <postgresql.version>42.3.5</postgresql.version>
-        <solr.version>8.11.1</solr.version>
-        <aws.version>1.11.762</aws.version>
-        <google.cloud.version>0.157.0</google.cloud.version>
+        <postgresql.version>42.5.0</postgresql.version>
+        <solr.version>8.11.2</solr.version>
```

Comment:
Hmm, does this mean we should also bump the version of Solr? Or is this just a problem with SolrJ?

Reply:
I think I ended up bumping Solr from my recent Docker container builds thus this bump.  I suspect they all should move together (Solr, SolrJ, etc.).  If you want me to take this one out let me know.

Reply:
@carlsonp just so I'm clear on this, you bumped Solr not for security reasons but because a new version is out?

I don't see any security issues at https://solr.apache.org/docs/8_11_2/changes/Changes.html . Just bug fixes. A lot of them. 27 bug fixes.

Since this PR relates to security, if the Solr bump in the pom is for bug fixes, let's please back out this one line.

Ideally when we bump the Solr line in the pom we also bump Solr itself. Doing a full Solr bump is a much bigger job with many more files to touch. If you feel like we should bump all of Solr for those 27 bug fixes, please feel free to create a dedicated issue for this. Thanks!

Reply:
Yep, that makes sense.  I've backed out Solr.